Simplenote/Classes/SPTagsListViewController.m

```diff
@@ -370,6 +370,9 @@ - (void)configureCell:(SPTagListViewCell *)cell atIndexPath:(NSIndexPath *)index
             [self configureBottomCell:cell atIndexPath:indexPath];
             break;
     }
+
+    cell.accessibilityLabel = cell.textField.text;
+    cell.textField.isAccessibilityElement = NO;
```

Comment:
Since this would essentially apply to every single instance, do you think it'd be best to make the cell perform these calls?

Reply:
Good point. Also this way can directly return `textField.text` from an `accessibilityLabel` accessor. 👍 
Thanks!